docs/.nav

```diff
@@ -4,16 +4,19 @@ scylla_specific
 execution_profiles
 lwt
 object_mapper
-geo_types
-graph
-graph_fluent
-classic_graph
 performance
 query_paging
 security
 upgrading
 user_defined_types
 dates-and-times
 cloud
+column-encryption
+geo_types
+graph
+classic_graph
+graph_fluent
+CHANGELOG
 faq
```

Comment:
Not sure if my previous comment was saved or not, so I'll repeat it.
Linking upstream changelog in our docs will imo be confusing for our users.
We should either write our own changelog, or remove this from docs.
Given that this PR is about merging upstream, I think it should remove changelog from nav.

Reply:
removed